sport/app/football/feed/Competitions.scala

```diff
@@ -329,8 +329,8 @@ object CompetitionsProvider {
     Competition("333", "/football/womens-fa-cup", "Women's FA Cup", "Women's FA Cup", "English"),
     Competition(
       "423",
-      "/football/womens-euro-2022",
```

Comment:
Do we need the starting / ? the other events seem to keep it

Reply:
Fixed :)